.github/dependabot.yml

```diff
@@ -57,15 +57,23 @@ updates:
     open-pull-requests-limit: 10
 
   - package-ecosystem: "gradle"
-    directory: "/packages/google_maps_flutter/google_maps_flutter/android"
+    directory: "/packages/google_maps_flutter/google_maps_flutter/example/android/app"
     commit-message:
       prefix: "[google_maps]"
     schedule:
       interval: "weekly"
     open-pull-requests-limit: 10
 
   - package-ecosystem: "gradle"
-    directory: "/packages/google_maps_flutter/google_maps_flutter/example/android/app"
+    directory: "/packages/google_maps_flutter/google_maps_flutter_android/android"
```

Comment:
Are these two examples apps now duplicated or are they different?

Reply:
They are basically the same; see https://github.com/flutter/flutter/issues/46716

Longer term it's something we want to fix, but we haven't been blocking federation on it. (It's a bit tricky to solve even beyond the package structure logistics, because sometimes, like here, we have shims that make the actual example code almost identical, whereas in others the examples are written against a different API layer; it depends on whether there is important widget code in the app-facing package. So we'll need a flexible approach.)